### DISCUSSION TITLE
关于亲密度未满后手动分享点赞亲密度仍不变的询问

### DISCUSSION THREAD

**Discussion by URGERKING** [andywang425/BLTH#267] [Q&A] [answered]
5.8.2连续使用了两天,个别勋章未满1200,处于1100/1000/900,**在手机上手动执行分享点赞等操作亲密度仍然不变**
请问这是被啊b风控了么🤧
详细说明:
正如开头所说,我有47个勋章,间隔设在0.5秒,脚本5.8.2,Tampermonkey 4.17.6161,Firefox99.
排除了不能发言的直播间,每天都会有个别勋章未满最大亲密度,问下大伙这是被风控了吗,但天选之类的都能正常参加...
另外我在一开始使用5.8.1时是能到1200的,所以我发现有些勋章不满后降级到了5.8.1重新执行任务但依然无变化🤧

**Comment by andywang425** [comment, marked as answer]
挂满30分钟，间隔时间拉长。大部分应该是能挂到1300的。

**Reply by URGERKING** [reply]
谢谢!我会在后面几天尝试不同设置的,不知道其他人有没有也像我一样...但是主要是这个情况比较令人困惑
>在手机上手动执行分享点赞等操作亲密度仍然不变

另外我没有算上30分钟的那个100亲密度,这里就以1200为最大亲密度吧
(已解决,磕头🙇‍‍♂️)

**Reply by letterk** [reply]
我也有这个问题, 12个牌子, 1100-1300不等, 心跳时间间隔应该如何正确设置?

**Reply by URGERKING** [reply]
> 我也有这个问题, 12个牌子, 1100-1300不等, 心跳时间间隔应该如何正确设置?

2秒朝上,可以在 #263 里讨论
